pkg/kubelet/kubeletconfig/util/codec/codec.go

```diff
@@ -50,8 +50,8 @@ func NewJSONEncoder(groupName string) (runtime.Encoder, error) {
 }
 
 // DecodeKubeletConfiguration decodes an encoded (v1alpha1) KubeletConfiguration object to the internal type
-func DecodeKubeletConfiguration(data []byte) (*componentconfig.KubeletConfiguration, error) {
-	// TODO(mtaufen): when KubeletConfiguration moves out of componentconfig, will the UniversalDecoder still work?
+func DecodeKubeletConfiguration(data []byte) (*kubeletconfig.KubeletConfiguration, error) {
```

Comment:
I think we want to make this look more like kube-proxy. Also this utility seems like it should go in cm/kubelet/server